purchases/src/main/kotlin/com/revenuecat/purchases/PostPendingTransactionsHelper.kt

```diff
@@ -23,12 +23,12 @@ internal class PostPendingTransactionsHelper(
 
     fun syncPendingPurchaseQueue(
         allowSharingPlayStoreAccount: Boolean,
-        onError: ((PurchasesError) -> Unit)? = null,
-        onSuccess: ((CustomerInfo?) -> Unit)? = null,
+        onError: ((PurchasesError, Boolean?) -> Unit)? = null,
+        onSuccess: ((CustomerInfo?, Boolean?) -> Unit)? = null,
```

Comment:
Is it needed to propagate whether there were unsynced purchases from here? I believe that if we call the `onSuccess` with `null`, that meant there were no unsynced purchases right? It might be good to make it more explicit, but don't know if we want the extra complexity for that... Might be worth just documenting the behavior as part of this method.

Reply:
> I believe that if we call the `onSuccess` with `null`, that meant there were no unsynced purchases right?

Not really. In this case there's a differentiation between
`true` --> there were unsynced purchases
`false` --> there were no unsynced purchases
`null` --> it was not checked whether or not there were unsynced purchases

> Is it needed to propagate whether there were unsynced purchases from here?

I really think so. This `syncPendingPurchaseQueue` method is where this information is fetched and used. So in the callbacks of this method we need to know not only the result (PurchasesError or CustomerInfo) but also whether there were unsynced purchases.